_locales/en/messages.json

```diff
@@ -49,6 +41,34 @@
         }
       }
     },
+    "importButton": {
+      "message": "Import",
+      "desription": "Button shown on the choose directory dialog which starts the import process"
+    },
+    "importChooserTitle": {
+      "message": "Choose directory with exported data",
+      "description": "Title of the popup window used to select data previously exported"
+    },
+    "importError": {
+      "message": "Unfortunately, something went wrong during the import. First, make sure all of the originally exported files are available. Then, please submit a debug log so we can help you get migrated!",
```

Comment:
Yeah, you're right, it would be good to give better guidance for people having trouble...